litmus/.ocamlformat

```diff
@@ -0,0 +1 @@
+disable = true
```

Comment:
I really like the idea of the -ignore files because it meant that any new file will be checked, should we do this for litmus, tools and jingle and internal as well?